store/rootmulti/store.go

```diff
@@ -250,15 +250,15 @@ func (rs *Store) loadVersion(ver int64, upgrades *types.StoreUpgrades) error {
 
 		store, err := rs.loadCommitStoreFromParams(key, commitID, storeParams)
 		if err != nil {
-			return errors.Wrap(err, "failed to load store")
+			return fmt.Errorf("failed to load store: %w", err)
```

Comment:
Why don't we want to wrap the error with `errorsmod.Wrap`? Seems like we lose useful info this way?

Reply:
What info do you lose? I took a look at that type and it doesn't buy us anything apart from a possible stacktrace, which can be handled.